### ISSUE TITLE
Broken edition render

### ISSUE THREAD

**Issue by tfmorris** [internetarchive/openlibrary#2231]
### Description

This returns the error

> /opt/openlibrary/deploys/openlibrary/e10c3f4/openlibrary/templates/type/edition/view.html: error in processing template: UnicodeEncodeError: 'ascii' codec can't encode characters in position 0-2: ordinal not in range(128) (falling back to default template)
> Unable to render this page.
> 

### Relevant url?
https://openlibrary.org/books/OL25434654M/eco
https://openlibrary.org/books/OL25434654M/eco?debug=true

This edition has an OCAID of ocaid: "嗯嗯嗯" which may be part of the problem, but we should handle the error better.

**Comment by LeadSongDog** [comment]
Google translates 嗯嗯嗯 as "Uh huh". The user https://openlibrary.org/people/xenos made only that one edit, the day after registering with the username lijiaqi "Now Archie". Smells like a prank.

**Comment by guyjeangilles** [comment]
Not seeing this error anymore
![#2231](https://user-images.githubusercontent.com/7256157/69013704-0e136180-097b-11ea-902f-16d982758f37.png)
"嗯嗯嗯" is still in the OCAID object. Closing.

{"publishers": ["Ceolfrith Arts Centre"], "classifications": {}, "created": {"type": "/type/datetime", "value": "2013-12-09T07:04:03.211044"}, "title": "Eco-aesthetics", "identifiers": {}, "languages": [{"key": "/languages/eng"}], "last_modified": {"type": "/type/datetime", "value": "2019-07-23T16:44:08.303358"}, "publish_date": "2002", "key": "/books/OL25434654M", "ocaid": "\u55ef\u55ef\u55ef", "publish_places": ["Sunderland, England"], "works": [{"key": "/works/OL16809839W"}], "type": {"key": "/type/edition"}, "latest_revision": 3, "revision": 3}

**Comment by tfmorris** [comment]
This problem is still reproducible. Apparently, which I didn't realize originally, it only happens for logged in users, so I'm guessing @guyjeangilles wasn't logged in when they attempted for reproduce it.

**Comment by guyjeangilles** [comment]
I am now seeing the error when I'm logged in. #579 required similar conditions to produce an error

**Comment by dcapillae** [comment]
Hi. I receive a similar message in the following record when I try to edit it:

https://openlibrary.org/books/OL21550998M/La_guerra_de_Vietnam_entre_la_historia_y_el_cine

There are two identical editions of the same book, one of them added by me. I didn't realize it already existed in the Open Library. I tried to edit the newly created record to delete it, but it won't let me edit it either. I can't edit either one, neither the original nor the duplicate.

https://openlibrary.org/books/OL28712980M/La_guerra_de_Vietnam_entre_la_historia_y_el_cine (duplicated)

**Comment by LeadSongDog** [comment]
This seemed to come up when editing editions by any author whose name contains ‘é’, ‘ü’ ‘ò’, and possibly many other non-ASCII characters. I verified this triggers the problem by removing the accents from OL73183A: the above editions were now editable. Merged them under OL9114439W. I left the accented spelling in the a.k.a. list, but that's obviously not the right long term answer.

**Comment by cdrini** [comment]
I can no longer reproduce this logged in or logged out.

**Comment by cdrini** [comment]
@dcapillae I was able to edit the record you mentioned as well.

**Comment by dcapillae** [comment]
Thanks, @cdrini. I think these issues (or some of them) were resolved by some changes that were made regarding the use of special characters in languages different than English (I don't remember the date of these changes). Now it works fine for me too. 

In the case of my edition, the issue was related to the author's name, "José María" (with two accents). I have tested and now it works correctly even with accents.
